### PR TITLE
I'am upgrade in ru-Ru.php

### PR DESCRIPTION
**Pull request by drycov**
I translate the russian slang an any words&termins

### CODE REVIEW

File: web/lang/ru_ru.php [hunk 1]
```diff
@@ -51,7 +50,7 @@
 // do this by default, uncomment this if required.
 //
 // Example
-header( "Content-Type: text/html; charset=utf-8" );
+header( "Content-Type: text/html; charset=koi8-r" );
```
Comment (SteveGilvarry) [inline web/lang/ru_ru.php:53]: Everything I read says utf-8 replaces koi8-r fine?

File: web/lang/ru_ru.php [hunk 2]
```diff
@@ -59,7 +58,7 @@
 // is subtlely different in versions of PHP before and after 4.3.0, see
 // http://uk2.php.net/manual/en/function.setlocale.php for details.
 // Also be aware that changing the whole locale may affect some floating point or decimal
-// arithmetic in the database, if this is the case change only the individual locale areas
+// arithmetic in the database, if this is the case change only the individual locale Зонаs
```
Comment (SteveGilvarry) [inline web/lang/ru_ru.php:61]: Find and replace is working too well, like this and including swapping "areas" into Russian in the middle of untranslated phrases.

**Comment by SteveGilvarry** [comment]
Thank you for taking the time to work on this, however I think you may have an older version of the content as you have more deletes than adds? Some new translations are removed and I know they are still valid lang strings from the GUI

File: web/lang/ru_ru.php (hunk 1 above)
Comment (ghost) [inline web/lang/ru_ru.php:53]: Yes , I use version 1.26 on Ubuntu 14.04 . And this translated text nekooryh action is obsolete in version 1.29 you use an encoding of UTF - 8 . Glad to help the project .
Yes sorry for uzhastny English . I just live in the Republic of Kazakhstan .

> Вторник,  1 марта 2016, 5:03 -08:00 от Steve Gilvarry notifications@github.com:
> 
> In  web/lang/ru_ru.php :
> 
> > @@ -51,7 +50,7 @@
> >  // do this by default, uncomment this if required.
> >  //
> >  // Example
> > -header( "Content-Type: text/html; charset=utf-8" );
> > +header( "Content-Type: text/html; charset=koi8-r" );
> > Everything I read says utf-8 replaces koi8-r fine?
> > —
> > Reply to this email directly or  view it on GitHub .

С уважением,
Denis Rycov
drycov@bk.ru

Comment (SteveGilvarry) [inline web/lang/ru_ru.php:53]: Denis if you want to update your 1.26 file to current github master version and then make some changes still you can still submit changes. I can see some of the changes are good.
Your english is better than my russian, so thank you for contributing.

**Comment by knnniggett** [comment]
I'm going to close this out due to inactivity. 

@drycov we will accept any future pull requests which incorporate the changes previously mentioned by @SteveGilvarry

**Pull request closed without merging**